docs/source/guide/get_started.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Label Studio
+title: Label Studio overview
```

Comment:
Are we using Title Case for titles?

Reply:
Sentence case. This one is right, the other one is wrong.